tools/cldr-apps/js/src/views/About.vue

```diff
@@ -1,5 +1,73 @@
 <template>
```

Comment:
comments:
- the CSS is *localized*.  Saves hunting around.
- this code simply enumerates the properties in `aboutData`.